capstone/src/main/resources/static/index.html

```diff
@@ -17,6 +17,7 @@
         <li class="active"><a href="index.html">Events</a></li>
         <li><a href="organizationsearch.html">Organizations</a></li>
         <li class=""><a href="profile.html">My Account</a></li>
+        <li><a href="map.html">Event Map</a> </li>
```

Comment:
It would truly be helpful to start using Thymeleaf. I am going to start working on it so that we don't have to manually change every navbar.